advection_test/tracerAdvection.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.                func.plot_projection_field(lon_global.data, lat_global.data, smoke_global[plotStep] - smoke_global[0], 
</pre><p>didn't you add a flag to this? Seems like it always plots the diff?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2cm/pace/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
The plotting should make two figures - one for the actual tracer, the second one for the difference from initial state.

Reply:
Removed the difference figure - now it only makes one